fixtures/container-app/README.md

```diff
@@ -1,3 +1,24 @@
 # HTTP fetch to a container
 
 This example shows a simple container setup where a DO passes requests through to a node server.
+
+## Simplified Container Configuration
+
+Only need to define the `[[containers]]` section in `wrangler.toml`. The Durable Object bindings and migrations are automatically generated:
```

Comment:
The PR mentions "wrangler.toml" at multiple location, could you please use "wrangler configuration" instead (i.e. line 9 below is jsonc)